Tensile/TensileBenchmarkCluster.py

```diff
@@ -0,0 +1,846 @@
+
+if __name__ == "__main__":
+    print("This file can no longer be run as a script.  Run 'Tensile/bin/TensileBenchmarkCluster' instead.")
+    exit(1)
+
+
+import shlex, subprocess
+import sys
+import os
+import argparse
+
+from .BenchmarkSplitter import BenchmarkSplitter
+from .Configuration import ProjectConfig
+from Tensile.Utilities.merge import mergePartialLogics
+
+try:
+    import mgzip
```

Comment:
`mgzip` seems to have the same interface as `gzip`.  This means we should be able to do:

``` python
        import mgzip as gzip
```

That way the usage below can be common to both libraries.  See how this is implemented with `tqdm` in `Utils.py`.

Reply:
Nice tip! Will implement.